keyboards/chickenman/ciel/readme.md

```diff
@@ -0,0 +1,21 @@
+# Ciel
+
+![img](https://i.imgur.com/8JlLbefl.png)
+
+The Ciel is a CNC'ed 60% Bakeneko with a brass internal weight.
+
+Group Buy ran October 7th to October 31st in 2021. Fulfilled early March.
+
+* Keyboard Maintainer: [Ramon Imbao](https://github.com/ramonimbao)
+* Hardware Supported: ATmega32u2
+* Hardware Availability: [GB page](https://geekhack.org/index.php?topic=114845.0), [ChickenMan's Discord](https://discord.gg/6MeZAxp2RP)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Added, thanks!